openedx/core/djangoapps/theming/storage.py

```diff
@@ -177,11 +177,11 @@ def _url(self, hashed_name_func, name, force=False, hashed_files=None):
         See the class docstring for more info.
         """
         processed_asset_name = self._processed_asset_name(name)
-        return super(ThemeCachedFilesMixin, self)._url(hashed_name_func, processed_asset_name, force, hashed_files)
+        return super(ThemeManifestFilesMixin, self)._url(hashed_name_func, processed_asset_name, force, hashed_files)
```

Comment:
Nit: since this is python 3 now, you should just call `super()` with no parameters.